provisioning/roles/wg/tasks/main.yml

```diff
@@ -0,0 +1,93 @@
+ ---
+- name: Add debian unstable repo
+  copy:
+    src: templates/unstable.list
```

Comment:
This template doesn't exist in the aarch64 repo yet, but it should be debian backports and not debian unstable iirc.